classes/Tools.php

```diff
@@ -3918,57 +3918,6 @@ public static function cleanNonUnicodeSupport($pattern)
         return $characterCleaner->cleanNonUnicodeSupport($pattern);
     }
 
-    protected static $is_addons_up = true;
-
-    public static function addonsRequest($request, $params = [])
```

Comment:
I think this method is still used in [`ps_themecusto`](https://github.com/PrestaShop/ps_themecusto/blob/dev/controllers/admin/AdminPsThemeCustoConfiguration.php#L534), maybe a PR on the module `ps_themecusto` should be made first?

Reply:
Thanks. I will do it this afternoon.

Reply:
@atomiix Done in PrestaShop/ps_themecusto#47